gpkit/nomials/nomial_math.py

```diff
@@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self._unsubbed)
 
+    def simplify_posy_ineq(self, exps, cs):
+        "Simplify a posy <= 1 by moving constants to the right side."
```

Comment:
Agree with that.

The confusion is that this method only makes sense when called with the `exps` and `cs` that arise in `self._gen_unsubbed` and `self.as_posyslt1`. maybe it should be private? If it's part of the user-facing API then I want to know what are valid inputs, and it doesn't seem to make sense to print `self` in the errors for arbitrary inputs.